### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by fragmental**
A few minor edits on line 35 and 39, added a bunch of double spaces for line breaks, and changed some ' (apostrophes?) to ` (grave accents/backticks?)

### CODE REVIEW

**Comment by paglias** [comment]
Thanks! The file is ok but the pull request should be made against **develop** branch not **master**. I can't edit it for you but I believe you can, at worst just close it and open another one

**Comment by fragmental** [comment]
I thought I was in develop when I made the pr.  Guess not.  I'm not sure how to change it to develop.  I could close this and open another, but all of the changes were very minor.

File: README.md [hunk 1]
```diff
@@ -11,74 +11,71 @@ Built using Angular, Express, Mongoose, Jade, Stylus, Grunt and Bower.
...
-
-1. Set up MongoDB.  Steps:
-  1.  Download the latest production release of MongoDB from: http://www.mongodb.org/downloads
-  1.  Extract the zip file to the desired application directory.  Example:  c:\apps\mongodb-win32-x86_64-2.4.6
-  1.  Rename the folder from mongodb-win32-x86_64-2.4.6 to mongodb
-  1.  Create a data\db directory under the application directory.  Example:  c:\apps\mongodb\data\db
-  1.  Start up MongoDB using the following command:
-  'c:\apps\mongodb\bin\mongod.exe --dbpath c:\apps\mongodb\data'
+1. [Fork the repository](https://help.github.com/articles/fork-a-repo) on your computer.  
+2. Checkout the **develop** branch where all the development happens:
```
Comment (lefnire) [inline README.md:15]: btw, 1. 1. 1. actually works in markdown (will render as 1. 2. 3.), meant to allow you to insert new items easily

**Comment by lefnire** [comment]
cool beans, will cross-check this in a bit

**Comment by fragmental** [comment]
I noticed that 1.1.1. works in markdown after I had already changed it.  Unless anyone objects, I'm just going to close this.

**Pull request closed without merging**